h/api/models/document.py

```diff
@@ -37,7 +37,7 @@ def title(self):
         titles = [m.value for m in self.meta if m.type == 'title']
 
         if titles:
-            return titles[0]
+            return titles[0][0]
```

Comment:
Had a look at the `DocumentMeta` model and I'm clearly wrong, but this seems messy, as well as prone to failure. What if a meta row exists but the value is an empty list

I assume what's supposed to be happening is that there is only supposed be one meta row for each document with type 'title' -- indeed I imagine that's enforced by a database constraint -- but as with all meta objects, we can have multiple values (`<title>`, DC metadata title, etc.).

It just needs to be a bit clearer that one of these arrays is to deal with the case that we have no title information, and the other is to deal with the fact that we can have multiple values for the title.